### PR TITLE
Website enhancements toggle

### PR DESCRIPTION
**Pull request by dylancom**
Can be toggled from settings.
Solves: #306

### CODE REVIEW

File: src/app/screens/Settings.tsx [hunk 1]
```diff
@@ -1,36 +1,109 @@
...
 function Settings() {
+  const [loading, setLoading] = useState(true);
+  const [settings, setSettings] = useState<{ websiteEnhancements: boolean }>({
+    websiteEnhancements: false,
+  });
   const [cameraPermissionsGranted, setCameraPermissionsGranted] =
     useState(false);
 
+  function getSettings() {
+    browser.storage.sync
```
Comment (bumi) [inline src/app/screens/Settings.tsx:36]: I think this should go through an `action` and the state. what do you think? 

in the end it is a `state.setState({})`

**Review by bumi — CHANGES_REQUESTED** [review]

File: src/app/screens/Settings.tsx (hunk 1 above)
Comment (dylancom) [reply src/app/screens/Settings.tsx:36]: Agree, will update :)

File: src/extension/content-script/batteries/index.ts [hunk 2]
```diff
@@ -8,6 +9,9 @@ import YouTubeVideo from "./YouTubeVideo";
 const enhancements = [Twitter, YouTubeVideo, Monetization];
 
 async function extractLightningData() {
+  const { settings } = await utils.call("getSettings");
+  if (!settings.websiteEnhancements) return;
```
Comment (bumi) [inline src/extension/content-script/batteries/index.ts:13]: just a thought for the future maybe: 
```
const { enabled }  = await utils.call("isFeatureEnabled", {feature: "websiteEnhancements"} );
```

**Review by bumi — APPROVED** [review]
utACK

(still need to test it)

**Pull request merged**